wp-config.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/** The name of the database for WordPress */
+define( 'DB_NAME', 'wordpress' );
+
+/** MySQL database username */
+define( 'DB_USER', 'wordpressuser' );
+
+/** MySQL database password */
+define( 'DB_PASSWORD', 'password' );
```

Comment:
I think this config file could be also converted into template and you woun't need to dublicate db_user, bassword, db_name here and during database creation

Reply:
yes, I'll change the wp-conf.php